pkg/operator/resources/operator/operator_test.go

```diff
@@ -0,0 +1,62 @@
+package operator
+
+import (
+	"fmt"
+	"github.com/RHsyseng/operator-utils/pkg/validation"
```

Comment:
Since this is explicitly imported, may want to explicitly add to glide.yaml with version info

Reply:
> Since this is explicitly imported, may want to explicitly add to glide.yaml with version info

added entry into glide.yaml